src/io/flutter/module/FlutterModuleBuilder.java

```diff
@@ -227,7 +227,7 @@ public String getBuilderId() {
   @Override
   @NotNull
   public ModuleType getModuleType() {
-    return FlutterModuleUtils.getModuleTypeForFlutter();
+    return ModuleTypeManager.getInstance().findByID(FlutterModuleUtils.getModuleTypeIDForFlutter());
```

Comment:
Any reason not to push this logic into `io.flutter.utils.FlutterModuleUtils#getModuleTypeForFlutter`?